src/test/java/hw6/exercise2/AssertWebElementsOnPageTest.java

```diff
@@ -0,0 +1,61 @@
+package hw6.exercise2;
+
+import hw6.BaseTest;
+import hw6.pages.JdiTestingDifferentElementsPage;
+import hw6.pages.JdiTestingHomePage;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+
+public class AssertWebElementsOnPageTest extends BaseTest {
+
+    @Test
+    public void HardAssertWebElements() {
```

Comment:
Why do you need this test in the current hw?

Reply:
Deleted